lib/ts/recipe/emailverification/emailVerificationClaim.ts

```diff
@@ -18,12 +22,15 @@ export class EmailVerificationClaimClass extends BooleanClaim {
 
         this.validators = {
             ...this.validators,
-            isVerified: (refetchTimeOnFalseInSeconds = 10, maxAgeInSeconds = 300) => ({
+            isVerified: (refetchTimeOnFalseInSeconds, maxAgeInSeconds) => ({
```

Comment:
Does this affect the func signature type? Earlier, both these inputs were optional (cause we assigned values to them). But now that we removed the default assignment, what's the input signature?

Reply:
It doesn't because the type comes from the `validators` property defined in this class.

<img width="1057" alt="image" src="https://github.com/supertokens/supertokens-web-js/assets/22813027/810625b2-ab49-479d-a6bb-6b22886cb84a">